support-workers/cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -207,6 +207,23 @@ Resources:
       Statistic: Sum
     DependsOn: SupportWorkersPROD
 
+    RevenueRecognitionSetupFailureAlarm:
+      Type: AWS::CloudWatch::Alarm
+      Condition: CreateProdResources
+      Properties:
+        AlarmActions:
+          - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
+        AlarmName: !Sub Revenue distribution for Digital Subscription gift failed in ${Stage}
```

Comment:
this is maybe not URGENT-9-5 but it definitely needs looking at as it is not self correcting when it goes back to OK like other alarms.

Reply:
What do you suggest? Including some wording to that effect?

Reply:
good question, maybe `DATA CORRUPTED - Revenue distribution for Digital Subscription gift failed in ${Stage}`  And there's a link to a doc that we can put more details in for each alarm, that usually goes in the description.
https://github.com/guardian/support-frontend/blob/920e638c35430cc260acdb1878f37bffa1d12fae/support-frontend/cloud-formation/cfn.yaml#L62

this is basically a data issue that will have been crystallised by this point.  The idea is to be clear to someone who doesn't know about that alarm whether they need to panic or what to do.

Ideally we would have a daily/weekly report of all redeemed gifts that don't have the right revenue distribution so they won't go away until they're fixed.  Maybe one for the new Data lake.